detox/test/src/Screens/DatePickerScreen.js

```diff
@@ -11,7 +11,7 @@ export default class DatePickerScreen extends Component {
     super(props);
 
     this.state = {
-      chosenDate: new Date(),
```

Comment:
That's a magic number, more semantic to use const here, e.g. `INITIAL_CHOSEN_DATE`